Havit.Blazor.Components.Web.Bootstrap/Tags/Internal/HxInputTagsInternal.razor.cs

```diff
@@ -111,6 +111,11 @@ public partial class HxInputTagsInternal
 		/// </summary>
 		[Parameter] public RenderFragment InputGroupEndTemplate { get; set; }
 
+		/// <summary>
+		/// Label to render before input.
```

Comment:
The `Label` here is not rendered before the input, as far as I see. Should we say "Label (only rendered for LabelType.Floating. The wrapping HxInputTags renders the label on his own for other cases)"? Or is it used in other scenarios?